### ISSUE TITLE
Clipboard support

### ISSUE THREAD

**Issue by l4l** [l4l/yofi#22]
It might be handy to support clipboard.

- Copy, as far as I know, won't work, because no other wayland client can get a buffer after exit. And since yofi is displayed as overlay no other window (except locking screen maybe) can get it. Therefore I don't feel like it make much sense to implement it.
- Paste should work fine.

**Comment by WhyNotHugo** [comment]
Copy would make sense if someone were using a clipboard manager.

However, there’s no functional clipboard manager out there, so that’s not really relevant **yet**.

**Comment by l4l** [comment]
Yeah, that's right, thanks for noticing. It seems there's one: https://github.com/yory8/clipman
